### PR TITLE
feat(create-session): [CHK-2071] add policy to create session `POST /sessions`

### PR DESCRIPTION
**Pull request by pietro-tota**
### List of changes
Add policy to create session token from wallet token using PM start session call

### Motivation and context
This policy implements create session token logic using PM start session api call

### Type of changes

- [x] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

### Other information

---

### CODE REVIEW

File: src/domains/ecommerce-app/api/ecommerce-io/v1/_create_new_session.xml.tpl [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            <return-response>
+                <set-status code="401" reason="Unauthorized" />
+                <set-body>
+                    @{
+                    JObject errorResponse = new JObject();
+                    errorResponse["title"] = "Unauthorized";
+                    errorResponse["status"] = 401;
+                    errorResponse["detail"] = "Unauthorized";
+                    return errorResponse.ToString();
+                    }
```
Comment (giovanniberti) [inline src/domains/ecommerce-app/api/ecommerce-io/v1/_create_new_session.xml.tpl:15]: Less C# 🙂 
```suggestion
                {
                    "status": 401,
                    "title": "Unauthorized",
                    "detail": "Unauthorized"
                }
```

File: src/domains/ecommerce-app/api/ecommerce-io/v1/_create_new_session.xml.tpl [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+         <return-response>
+                <set-status code="401" reason="Unauthorized" />
+                <set-body>
+                    @{
+                    JObject errorResponse = new JObject();
+                    errorResponse["title"] = "Unauthorized";
+                    errorResponse["status"] = 401;
+                    errorResponse["detail"] = "Unauthorized";
+                    return errorResponse.ToString();
+                    }
```
Comment (giovanniberti) [inline src/domains/ecommerce-app/api/ecommerce-io/v1/_create_new_session.xml.tpl:36]: Same here

File: src/domains/ecommerce-app/api/ecommerce-io/v1/_create_new_session.xml.tpl [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        </when>
+        <otherwise>
+         <return-response>
+                <set-status code="401" reason="Unauthorized" />
+                <set-body>
+                    @{
+                    JObject errorResponse = new JObject();
+                    errorResponse["title"] = "Unauthorized";
+                    errorResponse["status"] = 401;
+                    errorResponse["detail"] = "Unauthorized";
```
Comment (giovanniberti) [inline src/domains/ecommerce-app/api/ecommerce-io/v1/_create_new_session.xml.tpl:64]: Same here

Comment (pietro-tota) [reply src/domains/ecommerce-app/api/ecommerce-io/v1/_create_new_session.xml.tpl]: Fixed with https://github.com/pagopa/pagopa-infra/pull/1437/commits/9f13f77c039f797c0d30055abb4ee8b64ab5a134

File: src/domains/ecommerce-app/api/ecommerce-io/v1/_create_new_session.xml.tpl (hunk 2 above)
Comment (pietro-tota) [reply src/domains/ecommerce-app/api/ecommerce-io/v1/_create_new_session.xml.tpl]: Fixed with https://github.com/pagopa/pagopa-infra/pull/1437/commits/9f13f77c039f797c0d30055abb4ee8b64ab5a134

**Review by giovanniberti — DISMISSED** [review]

**Review by giovanniberti — APPROVED** [review]

**Pull request merged**